<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YHotKey</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>241</width>
      <height>193</height>
     </rect>
    </property>
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>OS</string>
     </property>
     <widget class="QRadioButton" name="Android">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>18</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Android</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="iOS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>18</y>
        <width>36</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>iOS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Samsung">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>18</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Samsung</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="settings_internet">
        <property name="text">
         <string>Настройки Internet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settings_mms">
        <property name="text">
         <string>Настройки MMS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="activation">
        <property name="text">
         <string>Активация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speedtest">
        <property name="text">
         <string>speedtest</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hello">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>5</y>
      <width>14</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="statistic">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>3</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>3</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>3</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>3</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="hello_Edit_2">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>3</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введи текст..</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Hi/Bye</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="helloButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hi</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboHi">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>20</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="byeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>46</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bye</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBye">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>55</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="add_Bye">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>55</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Добавить&lt;/b&gt; прощание</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_Bye">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>55</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Удалить&lt;/b&gt; выбранное прощание</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_Hi">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Добавить&lt;/b&gt; приветствие</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_Hi">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Удалить&lt;/b&gt; выбранное приветствие</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Другие шаблоны</string>
    </property>
    <widget class="QPushButton" name="few">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_few">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>20</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="add_few">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Добавить&lt;/b&gt; пользовательский шаблон</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_few">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;b&gt;Удалить&lt;/b&gt; выбранный пользовательский шаблон</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">ff0000</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="4" column="1">
       <widget class="QPushButton" name="b8">
        <property name="text">
         <string>Кнопка_8</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="b7">
        <property name="text">
         <string>Кнопка_7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="b6">
        <property name="text">
         <string>Кнопка_6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="b4">
        <property name="text">
         <string>Кнопка_4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="b2">
        <property name="text">
         <string>Кнопка_2</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="b1">
        <property name="text">
         <string>Кнопка_1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="b3">
        <property name="text">
         <string>Кнопка_3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="b5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Кнопка_5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>166</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Редактировать Текст</string>
        </property>
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Сохранить</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Old">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Вернуть первоначальное значение</string>
        </property>
        <property name="text">
         <string>Изначальный</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="constructor">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;b&gt;Добавить&lt;/b&gt; приветствие</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
